docs/source/recipes/deploying-elyra-with-opendatahub.md

```diff
@@ -101,9 +101,15 @@ spec:
         https://github.com/opendatahub-io/manifests/tarball/v1.0-branch-openshift
     - name: manifests
       uri: 'https://github.com/opendatahub-io/odh-manifests/tarball/master'
-  version: v0.7.0
+  version: v0.8.0
```

Comment:
The ODH community call this morning talked about ODH 0.9 with the Elyra image updates. Should we say 0.9 here?

Reply:
not yet since they dont actually have the 0.9.0 manifest out yet. 
Im actually not sure why they have this version number in there since it does not correspond with anything. The operator is sitting at version 1.0 and the odh-manifests do have a 0.8.0 release tag...HOWEVER...they always pull the master branch and not the tag, which is confusing